.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and deploy # Note: Deployment will be added in a future PR 
+
+on:
+  push:
+    branches-ignore:
+      - develop
+      - master
+      - support/**
+      - hotfix/**
+      - feature/**-i18n
+      - release/**
+
+jobs:
+  call_build:
+    uses: yext/answers-search-ui/.github/workflows/build.yml@dev/build-workflow
+
+  call_unit_test:
+    uses: yext/answers-search-ui/.github/workflows/unit_test.yml@dev/build-workflow
+    needs: call_build
+
+concurrency:
+  group: ci-build-and-deploy-${{ github.ref }}-1
```

Comment:
what's the `-1` for?

Reply:
I think that's something like a version number, similar to our `v3` in our circleci cache key

Reply:
It's just a version number that we can increment if we ever need to make a new group key for the same github ref. It's not really necessary but it also doesn't hurt to have it. It's similar to a cache key, and it's common to have a number on cache keys so that you can manually bust the cache.